### ISSUE TITLE
Run benchmarks on AWS

### ISSUE THREAD

**Issue by jreadey** [pangeo-data/storage-benchmarks#46]
Run benchmarks on AWS with a Kubernetes cluster.

**Comment by jreadey** [comment]
Hey @rsignell-usgs - Could @kaipak use your AWS cluster for running these benchmarks?

**Comment by rabernat** [comment]
Yes, this would be great.

Rich, we are eventually writing a paper about this, and we would be happy to include you as an author.

**Comment by kaipak** [comment]
Had a long conversation with @jreadey about this and other related topics last night and it should be relatively straightforward to run the tests in AWS with only minor changes in code. Not having to set up a new cluster for all this could potentially save a lot of time.

**Comment by rsignell-usgs** [comment]
Absolutely. I will invite @kaipak to a whitelisted org when I get to a computer.

**Comment by rsignell-usgs** [comment]
I have invited @kaipak to the whitelisted `cdi-workshops` org.

**Comment by kaipak** [comment]
Thanks much @rsignell-usgs
